conda/recipes/libcugraph/meta.yaml

```diff
@@ -34,29 +33,82 @@ build:
 
 requirements:
   build:
-    - cmake>=3.20.1,<3.23
-    - doxygen>=1.8.11
+    - cmake {{ cmake_version }}
+  host:
+    - doxygen {{ doxygen_version }}
     - cudatoolkit {{ cuda_version }}.*
-    - libraft-headers {{ minor_version }}
+    - libraft-headers {{ minor_version }}.*
     - libcugraphops {{ minor_version }}.*
     - librmm {{ minor_version }}.*
     - cudf {{ minor_version }}.*
-    - boost-cpp>=1.66
-    - nccl>=2.9.9
-    - ucx-proc=*=gpu
-    - gtest=1.10.0  # FIXME: pinned to version in https://github.com/rapidsai/integration/blob/branch-22.04/conda/recipes/versions.yaml
-    - gmock=1.10.0  # FIXME: pinned to version in https://github.com/rapidsai/integration/blob/branch-22.04/conda/recipes/versions.yaml
-  run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
-    - libraft-headers {{ minor_version }}
-    - librmm {{ minor_version }}
-    - nccl>=2.9.9
+    - boost-cpp {{ boost_cpp_version }}
+    - nccl {{ nccl_version }}
     - ucx-proc=*=gpu
-    - libcugraphops {{ minor_version }}.*
-    - libcusolver>=11.2.1
+    - gtest {{ gtest_version }}  # FIXME: pinned to version in https://github.com/rapidsai/integration/blob/branch-22.04/conda/recipes/versions.yaml
+    - gmock {{ gtest_version }}  # FIXME: pinned to version in https://github.com/rapidsai/integration/blob/branch-22.04/conda/recipes/versions.yaml
 
-about:
-  home: http://rapids.ai/
-  license: Apache-2.0
-  license_file: ../../../LICENSE
-  summary: libcugraph library
+outputs:
+  - name: libcugraph
+    version: {{ version }}
+    script: install_libcugraph.sh
+    build:
+      number: {{ GIT_DESCRIBE_NUMBER }}
+      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+    requirements:
+      build:
+        - cmake {{ cmake_version }}
+      run:
+        - cudatoolkit {{ cuda_spec }}
+        - libraft-headers {{ minor_version }}
+        - librmm {{ minor_version }}
+        - nccl {{ nccl_version }}
+        - ucx-proc=*=gpu
+        - libcugraphops {{ minor_version }}
+        - libcusolver {{ libcusolver_version }}
+    about:
+      home: http://rapids.ai/
+      license: Apache-2.0
+      license_file: ../../../LICENSE
+      summary: libcugraph library
+  - name: libcugraph_etl
+    version: {{ version }}
+    script: install_libcugraph_etl.sh
+    build:
+      number: {{ GIT_DESCRIBE_NUMBER }}
+      string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+    requirements:
+      build:
+        - cmake {{ cmake_version }}
+      host:
+        - {{ pin_subpackage('libcugraph', exact=True) }}
```

Comment:
Is this `host` requirement needed? I don't think it is since the `install_libcugraph_etl.sh` script is just running `cmake --install`